nextflow.config

```diff
@@ -237,8 +238,8 @@ profiles {
     test          { includeConfig 'conf/test.config'       }
     test_cache    { includeConfig 'conf/test_cache.config' }
     test_full     { includeConfig 'conf/test_full.config'  }
-    test_full_aws { 
-        includeConfig 'conf/test_full.config' 
+    test_full_aws {
+        includeConfig 'conf/test_full.config'
```

Comment:
Did this test file get implemented or is it for later?

Reply:
It's for later. I don't remember even adding this tbh...